src/language-html/parser-html.js

```diff
@@ -67,6 +67,19 @@ function normalizeAngularControlFlowBlock(node) {
   };
 }
 
+/**
+ * Converts an angular ICU expression to a text node so it can be printed as text.
+ */
+function convertAngularIcuExpressionToTextNode(node) {
+  if (node.type === "plural" || node.type === "select") {
+    node.type = "text";
+    node.value = node.sourceSpan.start.file.content.slice(
+      node.sourceSpan.start.offset,
+      node.sourceSpan.end.offset,
+    );
+  }
+}
```

Comment:
Why make it just a text?

Reply:
I wasn't familiar enough with the prettier codebase to print it correctly, what you've done in https://github.com/prettier/prettier/pull/15777 is a nicer way to solve it 😄 We can close this PR in favor of that one